### PR TITLE
Force lit-html dependency tree for correct types construction

### PR DESCRIPTION
**Pull request by Westbrook**
## What I did

1. Ensure that `lit-html@2.0` is the preferred version so that it can be depended on via bare module specifiers in types.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
     "http-server": "^13.1.0",
     "husky": "^7.0.4",
     "lint-staged": "^10.5.4",
+    "lit-html": "^2.0.0",
```
Comment (daKmoR) [inline package.json:87]: why is that needed? 🤔

Comment (Westbrook) [reply package.json:87]: Other wise lit-html@1.0 becomes the “default” due to Haunted, Lion and Rocket et al and the types get built as `lit/nodes_modules/lit-html` which isn’t guaranteed to be where that dependency is in other projects. This way it gets built at `lit-html` and can be resolved more naturally by consuming projects.

**Review by daKmoR — APPROVED** [review]
